### DISCUSSION TITLE
browser sync constantly refreshing

### DISCUSSION THREAD

**Discussion by luki-zz** [tailpress/tailpress#224] [General]
Hello, 
So I have this problem that browser sync is constantly refreshing, and I want to refresh only on save. How to fix it?

**Comment by jeffreyvr** [comment]
Try setting `injectChanges` to `true` here:

https://github.com/jeffreyvr/tailpress/blob/master/webpack.mix.js#L26

**Reply by luki-zz** [reply]
That didint solve the issue
